frontend/src/components/pages/ForgotPassword.tsx

```diff
@@ -0,0 +1,155 @@
+import React, { useState } from "react";
+import {
+  Box,
+  Input,
+  Text,
+  Button,
+  FormControl,
+  FormErrorMessage,
+  Flex,
+} from "@chakra-ui/react";
+import StatusMessage from "../common/StatusMessage";
+import background from "./login_background.png";
+import backgroundMobile from "./login_background_phone.png";
+
+type SentEmail = {
+  email: string;
+  timestamp: number;
+};
+
+const ForgotPassword = (): React.ReactElement => {
+  const [validUser, setValidUser] = useState(true);
+  const [sentEmail, setSentEmail] = useState(false);
+  const [sentEmailToUser, setSentEmailToUser] = useState(false);
+  const [userEmailId, setUserEmaild] = useState("");
+
+  const handleUserAuth = (userEmail: string) => {
+    const emailPattern = /^[^\s@]+@humanesociety\.org$/;
```

Comment:
maybe we should also include uwblueprint.org emails just for now for easy testing in the future